### ISSUE TITLE
The primary key does not consider the column name in the save

### ISSUE THREAD

**Issue by rxdrigocosta** [WendellAdriel/laravel-lift#14]
I'm using your wonderful library to handle a legacy database, and when I try to save a model it doesn't consider the column name when building the query.

My model look like this:
```php
<?php

namespace App\Models;

use Illuminate\Database\Eloquent\Model;
use WendellAdriel\Lift\Attributes\Column;
use WendellAdriel\Lift\Attributes\Config;
use WendellAdriel\Lift\Attributes\DB;
use WendellAdriel\Lift\Attributes\PrimaryKey;
use WendellAdriel\Lift\Lift;

#[DB(table: 'protocolo__setor', timestamps: false)]
final class Sector extends Model
{
    use Lift;
    
    #[PrimaryKey(type: 'string', incrementing: false)]
    #[Column(name: 'setor_id')]
    public string $id;

    #[Config(fillable: true, column: 'setor_descricao', rules: ['required', 'string'])]
    public string $description;

    #[Config(fillable: true, column: 'setor_visibilidade', rules: ['required', 'string'])]
    public string $visibility;

    #[Config(fillable: true, column: 'setor_email', rules: ['required', 'email'])]
    public string $email;

    #[Config(fillable: true, column: 'pes_id', rules: ['required', 'string'])]
    public string $person_id;
}
```

And when i do something like:
```php
$sector = Sector::first();
$sector->email = 'example@example.com';
$sector->save();
```

I get this error:

> "SQLSTATE[42S22]: Column not found: 1054 Unknown column 'id' in 'where clause' (Connection: instance, SQL: update `protocolo__setor` set `setor_email` = example@example.com where `id` is null)"

ps.: I accept suggestions on how to use your library to deal with legacy databases.

**Comment by WendellAdriel** [comment]
I see that you're using Column and PrimaryKey together and I didn't test this combination before, can you try updating your property to setor_id that I think it's the name of the column in the DB to see if it will work, please?

Also as a side note, for questions that we don't know yet the cause of or if it's a bug, better to use the Discussions instead of issues 😉

**Comment by rxdrigocosta** [comment]
Of course!

When changing the property to sector_id, no error is returned and the update is successful.

Sorry for the google translate english and misuse of issues ❤️

**Comment by rxdrigocosta** [comment]
@WendellAdriel Great, how should I proceed if I want a custom named property for my primary key?

**Comment by WendellAdriel** [comment]
@rxdrigocosta For now that's not supported, I added a note with that info in the docs.
The primary key can't have a custom column name defined for now.
But I'll take a look at it to see if I can make it work

**Comment by WendellAdriel** [comment]
@rxdrigocosta Tomorrow I'll test something out that I think will make this work! 💪

**Comment by rxdrigocosta** [comment]
Thank you very much!

I loved your library, I hope I can contribute with PR's, but first I need to study it a little kskskssk

**Comment by WendellAdriel** [comment]
@rxdrigocosta thank you so much! All contributions are great, you can contribute with the docs, with questions and answers and of course with PRs when you want to!

BTW, I just released v0.7.2 that should fix the issue with the primary key with custom name can you test it and let me know, please?
